site/content/3.11/develop/integrations/spring-boot-arangodb.md

```diff
@@ -25,39 +28,1476 @@ rules, i.e. minor or patch releases may introduce new features or breaking
 changes. Please refer to [releases](https://github.com/arangodb/spring-boot-starter/releases)
 for details.
 
-## Maven
+## Get started
+
+This is an extensive demo on how to use
+[Spring Data ArangoDB](https://github.com/arangodb/spring-data) with an example
+dataset of **Game of Thrones** characters and locations.
+
+### Build a project with Maven
+
+First, you have to set up a project and add every needed dependency.
+This demo use Maven and Spring Boot and adds `arangodb-spring-boot-starter` to
+auto-configure Spring Data ArangoDB.
 
-Add `arangodb-spring-boot-starter` to your project to auto-configure Spring
-Data ArangoDB.
+Create a Maven `pom.xml`:
 
 ```xml
-<dependency>
-  <groupId>com.arangodb</groupId>
-  <artifactId>arangodb-spring-boot-starter</artifactId>
-  <version>3.x-y</version>
-</dependency>
-```
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-## Configuration
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.1.3</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
 
-Configure the properties files of your application with the properties of
-[ArangoProperties](https://github.com/mpv1989/spring-boot-starter/blob/master/src/main/java/com/arangodb/springframework/boot/autoconfigure/ArangoProperties.java).
+    <groupId>com.arangodb</groupId>
+    <artifactId>spring-data-arangodb-tutorial</artifactId>
+    <version>1.0.0</version>
 
-```properties
-arangodb.spring.data.database=mydb
-arangodb.spring.data.user=root
-arangodb.spring.data.password=1234
+    <name>demo</name>
+    <description>Demo project for Spring Boot</description>
+
+    <properties>
+        <java.version>17</java.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.arangodb</groupId>
+            <artifactId>arangodb-spring-boot-starter</artifactId>
+            <version>3.1-0</version>
```

Comment:
Should we add comments here that users should check what versions are available and use the latest suitable version?